### DISCUSSION TITLE
生成英文目录

### DISCUSSION THREAD

**Discussion by Sunflower0423** [nju-lug/NJUThesis#247] [Q&A]
请问硕士的模版下怎么生成英文目录

**Comment by atxy-blip** [comment]
模板目前没有实现这个功能，请具体描述你的需求：比如，是在生成中文目录的同时额外生成一份包含英文章节标题的，还是仅仅把目录两个字改成英文。

**Reply by Sunflower0423** [reply]
是这样子我把“目录”已经试着改成英文了，现在是生成目录的时候 “第X章”需要改成英文的chapter
<img width="834" alt="截屏2024-03-19 20 08 34" src="https://github.com/nju-lug/NJUThesis/assets/164015846/6a0630e7-0aaa-4b32-9077-3832f6aa07cd">

**Reply by atxy-blip** [reply]
见 [ctex.pdf](http://mirrors.ctan.org/language/chinese/ctex/ctex.pdf) 7.1 节，在导言区加下面一段
```tex
\ctexset{
  chapter/number = \thechapter,
  chapter/name = {\chaptername\space},
  % section/name = {Section\space} % 可以类比修改
}
```
